ApollosProject/apollos-prototype
# Adds Person Birth Date

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds the ability to set a Person's birth date from the API. To be used in Burke's onboarding PR.

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

1. Boot up the API
2. Login as your user and store your auth token
3. Call the updateBirthDate mutation
```
mutation updateBirthDate {
  updateProfileFields(input:[{field:BirthDate, value:"2005-12-11T22:02:35.438Z"}]) {
    birthDate
  }
}
```
4. Check to make sure your birthDate is updated
```
query getCurrentUser{
  currentUser {
    profile {
      birthDate
    }
  }
}
```

### What automated tests did you write?

## TODO:

- [x] References #554 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
